src/styles/ftva/_section-wrapper.scss

```diff
@@ -1,4 +1,9 @@
 .ftva.section-wrapper {
+    .section-header {
+        // Set to zero to remove margin under Section Title
+        // Explore Other Series
```

Comment:
Just a note that this will impact all section-headers in FTVA, not just explore other series. As long as that is expected, this looks fine

Reply:
Serena  [12:38 PM](https://uclalibrary.slack.com/archives/D05ME368HCM/p1726861120684419)
yep!
[12:38]
wait
12:39
yes, should be ok